test/functional/cmdLineTests/jfr/jfrevents.xml

```diff
@@ -87,7 +87,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<output type="success" caseSensitive="yes" regex="no">jdk.YoungGenerationConfiguration</output>
 		<output type="failure" caseSensitive="yes" regex="no">jfr print: could not read recording</output>
 	</test>
-	<test id="test jfr system process - approx 30 seconds" platforms="aix.*,linux.*,osx.*,zos.*">
+	<test id="test jfr system process - approx 30 seconds" platforms="aix.*,linux.*,osx.*,win.*,zos.*">
```

Comment:
Now that all platforms are supported; please just remove the `platforms=` filter.